libraries/Filter/HarmonicNotchFilter.cpp

```diff
@@ -304,3 +311,5 @@ void HarmonicNotchFilterParams::save_params()
   instantiate template classes
  */
 template class HarmonicNotchFilter<Vector3f>;
+template class HarmonicNotchFilter<float>;
```

Comment:
this is only for the test, wouldn't it be better to put it in the test class?

Reply:
tried that, it doesn't build